distribution/docs/pom.xml

```diff
@@ -119,6 +121,42 @@
     </profiles>
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.codehaus.gmaven</groupId>
+                <artifactId>groovy-maven-plugin</artifactId>
+                <version>2.1</version>
+                <inherited>false</inherited>
+                <executions>
+                    <execution>
+                        <id>aggregate-deps</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>execute</goal>
+                        </goals>
+                        <configuration>
+                            <source>
+                                def lines = []
+                                new File('${highest-basedir}').traverse(type: groovy.io.FileType.FILES, nameFilter: ~/maven-deps.txt/) { file ->
+                                    for (line in file.readLines())
+                                        lines.add(line.trim())
+                                }
+                                def output = ""
+                                Outer:
+                                for (line in lines.unique().sort()) {
+                                    if (!(line in ${dependency-ignored-lines})) {
+                                        for (ignoredPackage in ${dependency-ignored-packages}) {
+                                            if (line.startsWith(ignoredPackage))
+                                                continue Outer
```

Comment:
❗️ and a `go-to`!?

Reply:
loop labels are not the same as go tos

Reply:
Agree, but the it's a semantically thin line 😄

Reply:
Since multiple people pointed this out as a smell (and we should be trying to hold our community code to some higher standard even if it's in groovy and only a build file) - can we work to rewrite this without the label if? I'm sure any of us would be happy to sit down and help if you need a hand with it.